components/o11y/base/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
I missed it before, but you would also have to add the production and stage overlays, since the ApplicationSet will look for them.

`components/o11y/staging/kustomization.yaml`
`components/o11y/production/kustomization.yaml`

both include `../base` in their resources list.

Reply:
fixed